examples/basic-tracer-node/package.json

```diff
@@ -5,20 +5,17 @@
   "description": "Example of using @opentelemetry/tracing in Node.js",
   "main": "index.js",
   "scripts": {
```

Comment:
Suggest we have just `start` which exports to both and remove all other scripts.

Reply:
done